spec/workspace-spec.coffee

```diff
@@ -777,7 +777,7 @@ describe "Workspace", ->
         })
         workspace2.deserialize(atom.workspace.serialize(), atom.deserializers)
         item = atom.workspace.getActivePaneItem()
-        expect(document.title).toBe "#{item.getTitle()} - #{atom.project.getPaths()[0]} - Atom"
+        expect(document.title).toMatch ///^#{item.getTitle()}\ \u2014\ #{atom.project.getPaths()[0]}///
```

Comment:
That's exactly it. On Windows/Linux, the title will end in "— Atom", on OS X it will not. We're verifying that the title _starts_ with the expected item title / project path combination.